rs/bitcoin/ckbtc/minter/src/address.rs

```diff
@@ -100,20 +99,30 @@ pub fn derivation_path(account: &Account) -> Vec<ByteBuf> {
 
 /// Returns a valid extended BIP-32 derivation path from an Account (Principal + subaccount)
 pub fn derive_public_key(ecdsa_public_key: &ECDSAPublicKey, account: &Account) -> ECDSAPublicKey {
```

Comment:
@ninegua: not directly related to this change, but do you know if we have some stability tests for this method `derive_public_key`? It seems quite crucial that the same input returns the same output, but so far a quick search through its callers didn't show any test. Maybe that's something worth adding in a follow-up PR, WDYT?